swift/OnnxRuntimeExtensionsTests/SwiftOnnxRuntimeExtensionsTests.swift

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import XCTest
+import Foundation
+@testable import OnnxRuntimeBindings
+@testable import OnnxRuntimeExtensions
+
+final class SwiftOnnxRuntimeExtensionsTests: XCTestCase {
+
+    let modelPath: String = Bundle.module.url(forResource: "decode_image", withExtension: "onnx")!.path
+
+    func testCreateSessionWithCustomOps() throws {
+        do {
+            let env = try ORTEnv(loggingLevel: ORTLoggingLevel.verbose)
+            let options = try ORTSessionOptions()
+            try options.setLogSeverityLevel(ORTLoggingLevel.verbose)
+            try options.setIntraOpNumThreads(1)
+            // Register Custom Ops library using function name
+            try options.registerCustomOps(usingFunction: "RegisterCustomOps")
```

Comment:
Guessing we can use the function name because the stub function in OrtExt.mm ensures it's available - which seems like a relatively nice usage. 

Out of interest if that wasn't the case, what would the code look like to register with a function pointer?

Reply:
> Guessing we can use the function name because the stub function in OrtExt.mm ensures it's available - which seems like a relatively nice usage.

yeah.

Reply:
one thing to consider, it may be confusing if we have two separate ways of registering ORT extensions custom ops (function name vs. function pointer) between the SPM package and the CocoaPods package. would this function name approach only work with the SPM package?